src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Objects;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Calendar dateFromDateFormat = changeDateFormat(dateFrom);
+        Calendar dateToDateFormat = changeDateFormat(dateTo);
```

Comment:
Consider using `LocalDate` instead of `Calendar` for date manipulation. This is recommended for better performance and readability.